features/guidance/dedicated-turn-roads.feature

```diff
@@ -687,7 +687,7 @@ Feature: Slipways and Dedicated Turn Lanes
 
        When I route I should get
             | waypoints | route             | turns                                      | locations |
-            | s,f       | sabc,ae,dbef,dbef | depart,fork slight right,turn right,arrive | s,a,e,f   |
+            | s,f       | sabc,ae,dbef,dbef | depart,turn straight,turn right,arrive     | s,a,e,f   |
```

Comment:
Does the input map need tweaking to use the dedicated turn road?

Reply:
Do you mean we should modify test input in order to have exact the same result as it used to be. To be honest I am not familiar with feature we test here, just changed it assuming that it is caused by slight differences in distance calculation - if you can elaborate I would really appreciate.

Reply:
Yes the guidance message to stay straight no longer matches the route path that takes the right-hand road.
Perhaps this reflects on the brittleness of the guidance code that minor distance changes are breaking it.